OSU College of Ag was referenced still somehow

tursynay opened the pull request:
Updated the last sentence so it does not mention College of Agricultural Sciences, Office of Diversity and Office of Chief of Technology.

I have checked that:

- [ ] all pages' content, images, and styles still appear correctly
- [ ] all links still work
- [ ] redirects (e.g `deploy-preview-XX--tislab-website.netlify.app/melzoom`) are still working

monicacecilia commented:
Closing in favour of #74, which contains this information.

monicacecilia commented:
For details, see https://github.com/tis-lab/tislab.org/pull/74/commits/4800fc84ebbd64116ee71fe4e449c6ea2de73732

tursynay committed:
OSU College of Ag was referenced still somehow

Updated the last sentence so it does not mention College of Agricultural Sciences, Office of Diversity and Office of Chief of Technology.
